src/snowcli/cli/project/schema.py

```diff
@@ -70,6 +70,9 @@ def as_fully_optional(self) -> RelaxedMap:
                 Optional("role"): Regex(IDENTIFIER),
                 Optional("name"): Regex(IDENTIFIER),
                 Optional("warehouse"): Regex(IDENTIFIER),
+                Optional("distribution", default="internal"): Regex(
+                    IDENTIFIER
+                ),  # FIXME
```

Comment:
Bool might be confusing because now you need to remember which boolean is attached to which distribution. I'd rather have the user spell it out if it means there are no surprises once the `snow app` commands are run.

Reply:
Yeah, that was my bad (real drive-by review).
You should use https://hitchdev.com/strictyaml/using/alpha/scalar/enum/